DuckDuckGo/FavoriteHomeCell.swift

```diff
@@ -86,21 +85,20 @@ class FavoriteHomeCell: UICollectionViewCell {
         self.link = link
         
         let host = link.url.host?.dropPrefix(prefix: "www.") ?? ""
-        iconLabel.text = "\(host.capitalized.first ?? " ")"
         
         isAccessibilityElement = true
         accessibilityTraits = .button
-        accessibilityLabel = "\(link.title ?? "")). \(UserText.favorite)"
-        
-        titleLabel.text = link.title
-        
-        iconImage.isHidden = true
-        iconLabel.isHidden = false
+        accessibilityLabel = "\(link.displayTitle ?? "")). \(UserText.favorite)"
         
+        titleLabel.text = link.displayTitle
         iconBackground.backgroundColor = host.color
-        useImageBorder(true)
+        
+        if let domain = link.url.host?.dropPrefix(prefix: "www."),
```

Comment:
Going to leave this as is because www is the only subdomain we can safely remove and domain could be different things to different people, like removing other subdomains, but that logic is tricky without using a lookup.